src/lib/components/composites/list/NamedList.svelte

```diff
@@ -97,11 +139,21 @@ This can be e.g. a search bar.
             <span class="text-hint italic">{emptyHint}</span>
         {:else}
             <ul class="scroll-box space-y-4 pb-1">
-                {#each loadedItems as item (keySelector(item))}
-                    <li>
-                        {@render listItemComponent?.(item)}
-                    </li>
-                {/each}
+                {#if groupSelector !== undefined}
+                    {#each Object.entries(groupBy(loadedItems, groupSelector)) as [key, values] (key)}
+                        <div class="space-y-1">
```

Comment:
```suggestion
                        <div class="space-y-1 mt-10">
```
What do you think about this?

Reply:
Good idea, but in my opinion 40px is bit too much. What do you think of the new solution?

Reply:
This is also fine. I guess that's a matter of taste in the end. But looks good as it is now.